lib/logstash/inputs/twitter.rb

```diff
@@ -121,4 +121,32 @@ def run(queue)
       retry
     end
   end # def run
+
+  protected
+  def process_a_tweet( tweet, queue)
+    if tweet.is_a?(Twitter::Tweet)
```

Comment:
I would prefer to not use "unless tweet.is_a" simply to avoid an indentation level because I expect that other cases will be handled in the future.